xptifw/samples/sycl_perf_collector/sycl_perf_collector.cpp

```diff
@@ -220,50 +221,91 @@ XPTI_CALLBACK_API void xptiTraceInit(unsigned int major_version,
     // 4. XPTI_IGNORE_LIST=piPlatformsGet,piProgramBuild
     // 5. XPTI_SIMULATION=10,20,50,100
     //
+
+    // Set up the Verbose output flag first so we can provide progress output of
+    // the collector
+    const char *VerboseFlag = std::getenv("XPTI_VERBOSE");
+    // Check the environment variable for verbose output and set the appropriate
+    // flag, if the variable is set
+    if (VerboseFlag) {
+      if (std::stoi(VerboseFlag) == 0)
+        ShowVerboseOutput = false;
+      else
+        ShowVerboseOutput = true;
+    } else {
+      ShowVerboseOutput = false;
+    }
+
+    const char *DebugFlag = std::getenv("XPTI_DEBUG");
+    // Check the environment variable for debug output and set the appropriate
+    // flag, if the variable is set
+    if (DebugFlag) {
+      if (std::stoi(DebugFlag) == 0)
+        ShowDebugInformation = false;
+      else {
+        ShowDebugInformation = true;
+        ShowVerboseOutput = true;
+      }
+    } else {
+      ShowDebugInformation = false;
+    }
+
     const char *ProfOutFile = std::getenv("XPTI_SYCL_PERF_OUTPUT");
-    if (ProfOutFile)
+    if (ShowVerboseOutput && ProfOutFile)
       std::cout << "XPTI_SYCL_PERF_OUTPUT=" << ProfOutFile << "\n";
     const char *StreamsToMonitor = std::getenv("XPTI_STREAMS");
-    if (StreamsToMonitor)
+    if (ShowVerboseOutput && StreamsToMonitor)
       std::cout << "XPTI_STREAMS=" << StreamsToMonitor << "\n";
     const char *ColorOutput = std::getenv("XPTI_STDOUT_USE_COLOR");
-    if (ColorOutput)
+    if (ShowVerboseOutput && ColorOutput)
       std::cout << "XPTI_STDOUT_USE_COLOR=" << ColorOutput << "\n";
     const char *FirstCallsToIgnore = std::getenv("XPTI_IGNORE_LIST");
-    if (FirstCallsToIgnore)
+    if (ShowVerboseOutput && FirstCallsToIgnore)
       std::cout << "XPTI_IGNORE_LIST=" << FirstCallsToIgnore << "\n";
     const char *SimulationOverheads = std::getenv("XPTI_SIMULATION");
-    if (SimulationOverheads)
+    if (ShowVerboseOutput && SimulationOverheads)
       std::cout << "XPTI_SIMULATION=" << SimulationOverheads << "\n";
     const char *CalibrationFlag = std::getenv("XPTI_CALIBRATE");
-    if (CalibrationFlag)
+    if (ShowVerboseOutput && CalibrationFlag)
       std::cout << "XPTI_SIMULATION=" << CalibrationFlag << "\n";
-    const char *VerboseFlag = std::getenv("XPTI_VERBOSE");
-    if (VerboseFlag)
+    if (ShowVerboseOutput && VerboseFlag)
       std::cout << "XPTI_VERBOSE=" << VerboseFlag << "\n";
+    if (ShowVerboseOutput && DebugFlag)
+      std::cout << "XPTI_DEBUG=" << DebugFlag << "\n";
+
+    // Check the environment variable for colored output and set the appropriate
+    // flag, if the variable is set
+    if (CalibrationFlag) {
```

Comment:
NIT: this and similar could be simplified to the following. It would be just more clear for code reading.
CalibrationRun = false; 
if (CalibrationFlag && std::stoi(CalibrationFlag) != 0)
  CalibrationRun = true;